src/views/main-menu.js

```diff
@@ -0,0 +1,49 @@
+import React, {Component} from 'react'
+import {Redirect} from 'react-router-dom'
+import ReactDOM from 'react-dom'
+
+class MainMenu extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      currentView: props.view,
+      ref: undefined,
+      toRender: <div className="btn btn-group w-100">
+        <button className="btn btn-secondary" onClick={() => {
+          if (this.state.currentView !== 'home') {
+            this.setState({
+              toRender: <Redirect to="/"/>
+            })
+            this.forceUpdate()
```

Comment:
I think setState does re-renders automatically.
Either way, forceUpdate should be avoided.